libportability-gfx/src/impls.rs

```diff
@@ -256,6 +319,147 @@ pub extern "C" fn gfxGetInstanceProcAddr(
         vkGetPhysicalDeviceSurfaceCapabilitiesKHR, PFN_vkGetPhysicalDeviceSurfaceCapabilitiesKHR => gfxGetPhysicalDeviceSurfaceCapabilitiesKHR,
         vkGetPhysicalDeviceSurfaceFormatsKHR, PFN_vkGetPhysicalDeviceSurfaceFormatsKHR => gfxGetPhysicalDeviceSurfaceFormatsKHR,
         vkGetPhysicalDeviceSurfacePresentModesKHR, PFN_vkGetPhysicalDeviceSurfacePresentModesKHR => gfxGetPhysicalDeviceSurfacePresentModesKHR,
+
+        vkDestroyDevice, PFN_vkDestroyDevice => gfxDestroyDevice,
```

Comment:
why are device functions in `gfxGetInstanceProcAddr`?

Reply:
Afaik, GetInstanceProcAddr has addr of all functions

Reply:
> Function pointers for all Vulkan commands can be obtained with the command

Reply:
Can we share the function gather code with GetDeviceProcAddr?